index.html

```diff
@@ -0,0 +1,165 @@
+                                <li class="social-media-icons"><button type="button" class="btn">css</button></li>
+                                <li class="social-media-icons"><button type="button" class="btn">javascript</button></li>
+                            </ul>
+                            <h5><button type="button" class="btn-b">See Project</button></h5>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <div class="card">
+                <div class="card__header">
+                    <img src="./images/Portfolio3.png" alt="card__image" class="card__image" width="600">
+                </div>
+                <div class="card__body">
+                    <h1 class="tonic">Tonic</h1>
+                    <h5><b>CANOPY</b> <span>&#183;</span> Back end Dev <span>&#183;</span> 2015</h5>
+                    <p class="paratext">A daily selection of privately personalised reads no accounts or signups required</p>
+                </div>
+                <div class="card__footer">
+                    <div class="user">
+                        <div class="user__info">
+                            <ul>
+                                <li class="social-media-icons"><button type="button" class="btn">html</button></li>
+                                <li class="social-media-icons"><button type="button" class="btn">css</button></li>
+                                <li class="social-media-icons"><button type="button" class="btn">javascript</button></li>
+
+                            </ul>
+
+                            <h5><button type="button" class="btn-b">See Project</button></h5>
+
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <div class="card">
+                <div class="card__header">
+                    <img src="./images/Portfolio4.png" alt="card__image" class="card__image" width="600">
+                </div>
+                <div class="card__body">
+                    <h1 class="tonic">Tonic</h1>
+                    <h5><b>CANOPY</b> <span>&#183;</span> Back end Dev <span>&#183;</span> 2015</h5>
+                    <p class="paratext">A daily selection of privately personalised reads no accounts or signups required</p>
+                </div>
+                <div class="card__footer">
+                    <div class="user">
+                        <div class="user__info">
+                            <ul>
+                                <li class="social-media-icons"><button type="button" class="btn">html</button></li>
+                                <li class="social-media-icons"><button type="button" class="btn">css</button></li>
+                                <li class="social-media-icons"><button type="button" class="btn">javascript</button></li>
+
+                            </ul>
```

Comment:
[ Required Changes ]
- [ ] Please stick to just `ul` and `li` to list the items. It's not required to place a button inside of the `li` tag
- [ ] Also please adjust the styles of these badges to match the Figma design
You Design
![image](https://user-images.githubusercontent.com/50941074/159134231-96ce8dae-f248-4339-aee6-bd48489960e1.png)

The Figma Design
![image](https://user-images.githubusercontent.com/50941074/159134262-6ece98d3-184e-4de1-9424-717859e14753.png)

Reply:
Done resolved